### ISSUE TITLE
Dropdown item showing a weird blue border

### ISSUE THREAD

**Issue by soichih** [bootstrap-vue/bootstrap-vue#1320]
![screenshot from 2017-11-08 14-29-29](https://user-images.githubusercontent.com/923896/32570218-80a32586-c491-11e7-85c5-925878f5bf9b.png)

How can I revert this to the default bootstrap 4 style (just gray background)?

**Comment by tmorehouse** [comment]
This border depends on which browser (and version) you are using.

What you are seeing is Chrome's _focus ring_ outline (Firefox has a more subtle focus ring styling), and the styling will be different if you are using buttons (b-dropdown-item-button) instead of links (b-dropdown-item).

Bootstrap-Vue implements the WAI-ARIA accessibility standards that assist keyboard and sight-impaired users.  No focus ring makes it impossible for keyboard only users to determine which drop-down item they have navigated to.

You can crate a style override to change the focus-ring style (the CSS `outline` property) if you wont have any keyboard only users using your site/app.

**Comment by soichih** [comment]
OK. What exactly should I do to fix this?  Something like this?

```
<style>
* {
    outline-style: none
}
</style>
```

Is there way to prevent bootstrap-vue from modifying the bootstrap 4 CSS?

**Comment by tmorehouse** [comment]
Bootstrap-Vue isn't modifying teh Bootstrap V4 CSS.  What you are seeing is the styling  defined by both the browser and Bootstrp V4 CSS (all we do is focus the item(s) in the menus)

You ca target dropdown's specifically by:
```css
.dropdown .dropdown-menu .dropdown-item:focus {
  outline: none;
}

**Comment by soichih** [comment]
I thought bootstrap-vue is adding the blue border because I don't see it under bootstrap v4 documentation
> https://getbootstrap.com/docs/4.0/components/dropdowns/

![screenshot from 2017-11-09 09-04-33](https://user-images.githubusercontent.com/923896/32609302-3fdf3d72-c52d-11e7-8401-56f3fd5670f3.png)

**Comment by mosinve** [comment]
BS4 actually has this blueborder too, but only if focused by keyboard.

**Comment by tmorehouse** [comment]
For W3C WAI-ARIA accessibility, the recommendation is that the first first item in the menu is to receive focus when the menu is first opened.
